websites/P/Philo/dist/metadata.json

```diff
@@ -0,0 +1,21 @@
+{
+  "author": {
+    "name": "Kedi",
+    "id": "238668511222693888"
+  },
+  "service": "Philo",
+  "description": {
+    "en": "Live & on-demand TV. 60+ channels. Unlimited DVR. No contract."
+  },
+  "url": ["www.philo.com", "philo.com"],
+  "version": "1.0.0",
+  "logo": "https://i.imgur.com/Np5TVJB.png",
+  "thumbnail": "https://i.imgur.com/R9ivBkz.jpg",
+  "color": "#0C93F2",
+  "tags": [
+    "philo",
```

Comment:
idk why u added it to tags since no one would search philo as tag.

Reply:
Saw it like that in another presence so I figured I'd put it